### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by JinHai-CN**
### What problem does this PR solve?

1. Remove unused code
2. Fix type mismatch, in nlp search and infinity search interface
3. Fix chunk list, get all chunks of this user.

### Type of change

- [x] Bug Fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: api/apps/chunk_app.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def list_chunk():
         }
         if "available_int" in req:
             query["available_int"] = int(req["available_int"])
-        sres = settings.retrievaler.search(query, search.index_name(tenant_id), kb_ids, highlight=True)
+        sres = settings.retrievaler.search(query, [search.index_name(tenant_id)], kb_ids, highlight=True)
```
Comment (yuzhichang) [inline api/apps/chunk_app.py]: Not necessary.

File: api/apps/chunk_app.py [hunk 2]
```diff
@@ -307,7 +307,7 @@ def knowledge_graph():
         "doc_ids": [doc_id],
         "knowledge_graph_kwd": ["graph", "mind_map"]
     }
-    sres = settings.retrievaler.search(req, search.index_name(tenant_id), kb_ids)
+    sres = settings.retrievaler.search(req, [search.index_name(tenant_id)], kb_ids)
```
Comment (yuzhichang) [inline api/apps/chunk_app.py]: Not necessary.

File: rag/utils/infinity_conn.py [hunk 3]
```diff
@@ -249,28 +252,31 @@ def search(
...
+                                matchExpr.embedding_data_type,
+                                matchExpr.distance_type,
+                                matchExpr.topn,
+                                matchExpr.extra_options,
+                            )
+                        elif isinstance(matchExpr, FusionExpr):
+                            builder = builder.fusion(
+                                matchExpr.method, matchExpr.topn, matchExpr.fusion_params
+                            )
+                else:
```
Comment (yuzhichang) [inline rag/utils/infinity_conn.py:280]: What if `filter_cond` is empty?

File: api/apps/chunk_app.py (hunk 2 above)
Comment (JinHai-CN) [reply api/apps/chunk_app.py]: beartype will report error

File: api/apps/chunk_app.py (hunk 1 above)
Comment (JinHai-CN) [reply api/apps/chunk_app.py]: beartype will report error

**Pull request merged**